ui/core/src/store/poolFinder.ts

```diff
@@ -0,0 +1,20 @@
+import { Ref, toRefs } from "@vue/reactivity";
+import { Store } from ".";
+import { Asset, Pool } from "../entities";
+
+type PoolFinderFn = (
+  s: Store
+) => (a: Asset | string, b: Asset | string) => Ref<Pool> | null;
+
+export const createPoolFinder: PoolFinderFn = (s: Store) => (
```

Comment:
What does this find? A quick comment would be helpful imo

Reply:
It finds pools by asset pair in the store. I wanted to try pull it out of the setup code but because of the reactive store putting it on the store is not practical in terms of types as methods is not reactive data.